arch/risc-v/src/esp32c3/Kconfig

```
@@ -283,4 +297,84 @@ endif # ESP32C3_UART1
 
 endmenu
 
+menu "Real-Time Timer"
+	depends on ESP32C3_RT_TIMER
+
+config ESP32C3_RT_TIMER_TASK_NAME
+	string "Timer task name"
+	default "rt_timer"
+
+config ESP32C3_RT_TIMER_TASK_PRIORITY
+	int "Timer task priority"
+	default 223 # Lower than high priority workqueue
+
+config ESP32C3_RT_TIMER_TASK_STACK_SIZE
+	int "Timer task stack size"
+	default 2048
+
+endmenu # Real-Time Timer
+
+menu "WiFi configuration"
+	depends on ESP32C3_WIRELESS
+
+config ESP32C3_WIFI_STATIC_RXBUF_NUM
+    int "WiFi static RX buffer number"
```

```suggestion
      int "WiFi static RX buffer number"
```
nit: Alignment issue